snippets/csharp/System.Globalization/NumberFormatInfo/CurrencyNegativePattern/currencynegativepattern1.cs

```diff
@@ -48,4 +48,5 @@ public int Compare(CultureInfo x, CultureInfo y)
 //       de-LI:           2 (  $-n)       CHF-16.34
 //       de-LU:           8 ( -n $)        -16,34 €
 //       dsb-DE:          8 ( -n $)        -16,34 €
+//       luy-KE:         16 ( $- n)      Ksh- 16.34
```

Comment:
note, the result shown here happen only if using ICU. Anyone using old NLS legacy mode can get different results (like .NET Framework apps). Also, globalization data possibly can change which can make these results change too. 

Can we add a comment clarifying these two points in this code?